### DISCUSSION TITLE
Is it possible to set the logo for Coinbase wallet connect?

### DISCUSSION THREAD

**Discussion by JorensM** [rainbow-me/rainbowkit#1567] [Q&A] [answered]
Hello, I would like to change the logo that appears when connecting with Coinbase. Is that possible?

<img width="211" alt="Screenshot 2023-08-14 at 15 51 04 (2)" src="https://github.com/rainbow-me/rainbowkit/assets/17122123/7ef8975b-b0f0-47b1-86de-7089f10becc5">

Thanks!

**Comment by 0xmkh** [comment, marked as answer]
@JorensM We are going to introduce this in RainbowKit v2 which is coming out really soon

**Reply by JorensM** [reply]
Ok, thanks, sorry by the way about the duplicate post, I had forgotten about the first one when I posted the second one haha
